os/include/tinyara/wqueue.h

```diff
@@ -387,7 +387,7 @@ int work_lpstart(void);
  *
  ****************************************************************************/
 
-int work_queue(int qid, FAR struct work_s *work, worker_t worker, FAR void *arg, uint32_t delay);
+int work_queue(int qid, FAR struct work_s *work, worker_t worker, FAR void *arg, clock_t delay);
```

Comment:
If you change the input type of existing API, it might occur compile warnings. Am I right?

Reply:
I think there will be no compile  warning because clock_T is uint32_t or uint64_t